disgo.go

```diff
@@ -0,0 +1,22 @@
+// Package disgo is a logging and prompting library for
+// modern command line interfaces.
+//
+// It is NOT a substitute for a structured logger
+// in an application or a microservice, since it is
```

Comment:
Heh, a microservice is an application :p

Reply:
That's right, will edit this out :p